docs/schema/aws.md

```diff
@@ -650,6 +650,9 @@ Representation of an AWS [EC2 Key Pair](https://docs.aws.amazon.com/AWSEC2/lates
 | keyfingerprint | The fingerprint of the public key |
 | region| The AWS region |
 | **arn** | AWS-unique identifier for this object |
+ id | same as `arn` |
+| uploaded | `uploaded` is set to `True` if the the KeyPair was uploaded to AWS. Uploaded KeyPairs$
+| duplicate_keyfingerprint | `duplicate_keyfingerprint` is set to `True` if it's `keyfingerprint` $
```

Comment:
Finish the rest of this description, please update it to something like:

`duplicate_keyfingerprint` is set to `True` if this KeyPair has the same keyfingerprint as another one.